tests/data/test2307

```diff
@@ -31,6 +31,7 @@ upgrade
 <features>
 debug
 ws
+!hyper
```

Comment:
Can you please add this test to `DISABLED` for hyper instead? It makes it easier to get an overview of tests that are disabled for hyper builds.

Reply:
Added.